ami/models/model_names.py

```diff
@@ -11,3 +11,5 @@ class ModelNames(str, Enum):
     I_JEPA_PREDICTOR = "i_jepa_predictor"
     I_JEPA_CONTEXT_ENCODER = "i_jepa_context_encoder"
     I_JEPA_TARGET_ENCODER = "i_jepa_target_encoder"  # Always alias to other key: `IMAGE_ENCODDER`.
+    I_JEPA_CONTEXT_DECODER = "i_jepa_context_decoder"
+    I_JEPA_TARGET_DECODER = "i_jepa_target_decoder"
```

Comment:
`I_JEPA_CONTEXT_ENCODER`に対して`I_JEPA_CONTEXT_DECODER`としてしまうと、なんかVAEみたいにI-JEPAがDecoderを学習に必要とするモデルみたいなあらぬ意図が伝わってしまったりしないかな
あくまで可視化のみが目的のモジュールって意味合いを強調して、こんな感じの名前を代替案として提案したい
```suggestion
    I_JEPA_CONTEXT_VISUALIZATION_DECODER= "i_jepa_context_visualization_decoder"
    I_JEPA_TARGET_VISUALIZATION_DECODER = "i_jepa_target_visualization_decoder"
```

Reply:
OK